apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -339,6 +493,34 @@ parse_vtep_mac(MAC) ->
         end,
         MACComponents).
 
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, mesos_state_polls_total},
+        {help, "Total number of times Mesos overlay state has beed polled."}]),
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, mesos_state_poll_errors_total},
+        {help, "Total number of errors while polling Mesos overlay state."}]),
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, networks_added_total},
```

Comment:
Could you please add a gauge metric for a number of overlay networks instead of `networks_added_total`?